media/gpu/vaapi/vaapi_jpeg_decode_accelerator.cc

```diff
@@ -47,6 +47,9 @@ static void ReportToUMA(VAJDADecoderFailure failure) {
 // macro in the header file without pulling in the entire <va/va.h>.
 static_assert(VA_FOURCC_YUYV == VA_FOURCC('Y', 'U', 'Y', 'V'),
               "VA_FOURCC_YUYV must be equal to VA_FOURCC('Y', 'U', 'Y', 'V')");
+#ifndef VA_FOURCC_I420
```

Comment:
Commit message: where does this patch come from?

Reply:
We were having a build issue due a missing define for using an old libva version. I could check that OpenSUSE which was also including the VA-API changes was also needing to patch that, so it's imported from there https://build.opensuse.org/package/view_file/openSUSE:Factory/chromium/chromium-old-libva.patch?expand=1
I'm also making it explicit on the commit message.